templates/api/knowledge-panels/report_problem/incomplete_or_incorrect_data.tt.json

```diff
@@ -0,0 +1,30 @@
+{
+    "level": "info",
+    "topics": [
+        "problem"
+    ],
+    "expand_for": "large",
+    "evaluation": "neutral",
+    "title_element": {
+        "title": "[% edq(lang('incomplete_or_incorrect_data_title')) %]",
+        "subtitle": "[% edq(lang('incomplete_or_incorrect_data_subtitle_off')) %]",
+        "icon_url": "[% static_subdomain %]/images/logos/off-logo-icon-light.svg",
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "html": `
+                [% edq(lang('incomplete_or_incorrect_data_content_correct')) %]
+                [% edq(lang('incomplete_or_incorrect_data_content_correct_off')) %]
```

Comment:
edq here is not necessary.
We should instead escape backtick !

Reply:
right, thanks